src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature

```diff
@@ -12,30 +12,19 @@ Scenario: Solution Details
 	And Solution Full Description
 	And About Solution URL
 	And Contact Details
-	And Last Updated Date
 	And list of Capabilities
 	And Solution ID
 
-Scenario: Attachment Download Link Displayed
```

Comment:
where is this covered if you're removing it?

Reply:
DAPI test task added to backlog